src/pubsub/mod_pubsub_db.erl

```diff
@@ -79,7 +79,7 @@
 -callback find_nodes_by_key(Key :: mod_pubsub:hostPubsub() | jid:ljid()) ->
     [mod_pubsub:pubsubNode()].
 
--callback delete_node(Key :: mod_pubsub:hostPubsub() | jid:ljid(), Node :: mod_pubsub:nodeId()) -> ok.
+-callback delete_node(Node :: mod_pubsub:pubsubNode()) -> ok.
```

Comment:
That's actually for planned changes to RDBMS backend. I was thinking that in case of RDBMS we can delete the node by the `id` field which will be the primary key in `pubsub_nodes` table. The mnesia backend needs `nodeid` because that's the primary key for `pubsub_node` table in mnesia. If you that's not worthy I can change to use `nodeid` for both backends.